sdks/java/extensions/sql/build.gradle

```diff
@@ -206,6 +206,14 @@ task runBasicExample(type: JavaExec) {
   args = ["--runner=DirectRunner"]
 }
 
+// Run basic SQL example
+task runNestedRowInArrayExample(type: JavaExec) {
+  description = "Run basic SQL example"
+  mainClass = "org.apache.beam.sdk.extensions.sql.example.BeamSqlUnnestExample"
+  classpath = sourceSets.main.runtimeClasspath
+  args = ["--runner=DirectRunner"]
+}
```

Comment:
Do we need this example? It looks like the same behavior is verified continuously in [BeamSqlDslUnnestRowsTest.java](https://github.com/apache/beam/pull/16988/files#diff-8c0c041856075a2f10a667e185965b258fb8dd695a9a7ab9e4fdb417499fd424)

Reply:
Removed the example.